webapp/public/run.html

```diff
@@ -22,7 +22,6 @@
             position: relative;
             background: transparent;
             width: 100%;
-            padding-bottom: 81.96%;
```

Comment:
does this get injected later on? or is it useless?

Reply:
It gets set on the element. No idea why it was also specified in the page because it always gets ignored.